js-exercises/negative-index/negativeIndex.js

```diff
@@ -0,0 +1,30 @@
+const handler = {
+  get(target, prop) {
+    if (prop in target) {
+      return target[prop];
```

Comment:
I don't think this check is required. Can you help me understand?
In my opinion only two checks are required
1. `if (typeof property !== "string")`
2. `if not string then convert to number and check if isNaN true or false`. For more info please check my other comment.